### ISSUE TITLE
The new terminal creator shortcut to wsl doesn't work

### ISSUE THREAD

**Issue by betomaki** [microsoft/vscode#122498]
Issue Type: <b>Bug</b>

When I try to create a new terminal from dropbox in terminal called "New Ubuntu-20.04 (Padrão)", it fails:
"The terminal process "C:\WINDOWS\System32\wsl.exe '-d', 'Ubuntu-20.04 (Padrão)'" failed to launch (exit code: 4294967295)."
But I saw a flashed message saying: "Não há distribuição com o nome fornecido."

VS Code version: Code 1.55.2 (3c4e3df9e89829dce27b7b5c24508306b151f30d, 2021-04-13T09:35:57.887Z)
OS version: Windows_NT x64 10.0.19042

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-8550U CPU @ 1.80GHz (8 x 1992)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: enabled<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.90GB (17.03GB free)|
|Process Argv|--crash-reporter-id 40b25d93-0e2d-4c9f-8930-a858091ab5eb|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (21)</summary>

Extension|Author (truncated)|Version
---|---|---
vscode-sqlite|ale|0.12.0
mongodb-vscode|mon|0.5.0
vscode-docker|ms-|1.12.1
python|ms-|2021.4.765268190
vscode-pylance|ms-|2021.4.2
jupyter|ms-|2021.5.745244803
remote-containers|ms-|0.166.1
remote-ssh|ms-|0.65.4
remote-ssh-edit|ms-|0.65.4
remote-wsl|ms-|0.54.6
vscode-remote-extensionpack|ms-|0.20.0
cmake-tools|ms-|1.7.1
cpptools|ms-|1.3.1
hexeditor|ms-|1.5.0
powershell|ms-|2021.2.2
debugger-for-chrome|msj|4.12.12
vetur|oct|0.33.1
vue-vscode-snippets|sdr|2.2.0
cmake|twx|0.0.17
lorem-ipsum|Tyr|1.2.0
python-auto-venv|whi|1.2.0

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
pythonvspyt602:30294772
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstry914:30276682
pythonvsnew554cf:30291488
pythontb:30283811
vspre833cf:30267465
pythonptprofiler:30281270
vshan820:30294714
pythondataviewer:30285071
vscus158:30286553

```

</details>

**Comment by Tyriar** [comment]
I believe this is already fixed, can you see if you can reproduce in the [Insiders build](https://code.visualstudio.com/insiders)?

**Comment by betomaki** [comment]
Hi!
Awesome! It worked in the Insiders build and fantastic! The new look is pretty nice!
Thanks Daniel,

Best regards,

Humberto



    Em quarta-feira, 28 de abril de 2021 10:49:27 BRT, Daniel Imms ***@***.***> escreveu:  
 
 


Closed #122498.
